code/game/objects/items/robot/items/generic.dm

```diff
@@ -15,10 +15,17 @@
 /obj/item/borg/stun
 	name = "electrically-charged arm"
 	icon_state = "elecarm"
+	var/stamina_damage = 60 //Same as normal batong
 	/// Cost to use the stun arm
-	var/charge_cost = 1000
+	var/charge_cost = 200
+	var/cooldown_check = 0
+	/// cooldown between attacks
+	var/cooldown = 4 SECONDS // same as baton
 
 /obj/item/borg/stun/attack(mob/living/attacked_mob, mob/living/user)
+	if(cooldown_check > world.time)
+		to_chat(user, span_warning("The arm is still recharging!"))
```

Comment:
```suggestion
		user.balloon_alert(user, "still recharging!")
```
Unless you think there's a strong reason for this to be logged to the player for a long time, this seems like a pretty ephemeral piece of feedback.